Sources/Network/Pinning/ServerTrustEvaluator.swift

```diff
@@ -0,0 +1,381 @@
+//
+//  ServerTrustEvaluator.swift
+//  Alicerce
+//
+//  Created by André Pacheco Neves on 14/05/2018.
+//  Copyright © 2018 Mindera. All rights reserved.
+//
+
+import Foundation
+import Security
+
+// Heavily inspired by https://github.com/datatheorem/TrustKit 🙏
+
+public final class ServerTrustEvaluator {
+
+    // tag used to add and find the public key in the Keychain
+    static let keychainPublicKeyTag = "com.mindera.alicerce.ServerTrustEvaluator.keychainPublicKeyTag"
```

Comment:
Excellent point!

However, I think we should be safe, because for iOS apps to share Keychain entries they have to explicitly belong to a group. [source](https://developer.apple.com/documentation/security/keychain_services/keychains)

> In iOS, apps have access to a single keychain (which logically encompasses the iCloud keychain). This keychain is automatically unlocked when the user unlocks the device and then locked when the device is locked. An app can access only its own keychain items, or those shared with a group to which the app belongs. It can't manage the keychain container itself.

Besides, for a key to be accessible in a group it has to be defined using the `kSecAttrAccessGroup`, which we're not defining.

Lastly, the need for using the Keychain is only required for iOS 9 (to retrieve public key data). Once we move to iOS 10+ we can remove all this Keychain code 😄